frontend/src/components/eventselect/AnalyticsType.ts

```diff
@@ -62,6 +62,14 @@ export interface IEventAnalyticsItem {
 export interface IRetentionAnalyticsItem {
   startEventOption: IAnalyticsItem | null;
   revisitEventOption: IAnalyticsItem | null;
+  startConditionOptions: CategoryItemType[];
+  startConditionList: IConditionItemType[];
+  startConditionRelationShip: ERelationShip;
+  revisitConditionOptions: CategoryItemType[];
+  revisitConditionList: IConditionItemType[];
+  revisitConditionRelationShip: ERelationShip;
+  startEventRelationAttributeOptions: CategoryItemType[] | null;
```

Comment:
can we change to `startEventRelationAttributeOptions?: CategoryItemType[]`

Reply:
The upstream front-end components define this type.